### PR TITLE
Added Standardized College Codes and Applied them to PDF Export and Progress Tracker Toolcards

### PR DESCRIPTION
**Pull request by jerrry1123**
### Summary 

- Created a map to tidy up various college codes and standardized them
- Applied this map to college codes in the PDF export and progress tracker toolcards

### Test Plan 

- Create courses and apply them to requirements in the plan section, and then make sure that the college code in the toolcard and the college bubbles in the export PDF match with the map provided in Dashboard.vue.

### CODE REVIEW

**Review by zachary0kent — COMMENTED** [review]
Why is `collegeNames` passed around as a prop and a parameter? Shouldn't it be a constant?

**Comment by mirandayu131** [comment]
I like the college name map. Do we also need to include SHA codes for hotelies?

File: src/containers/Dashboard.vue [hunk 1]
```diff
@@ -159,6 +159,24 @@ export default defineComponent({
...
+        ['AG', 'CALS'],
+        ['EN', 'CoE'],
+        ['eng', 'CoE'],
+        ['HE', 'HumEc'],
+        ['humec', 'HumEc'],
+        ['IL', 'ILR'],
+        ['ilr', 'ILR'],
+        ['johnson', 'Dyson'],
+        ['BU', 'Dyson'],
+      ]),
```
Comment (zachary0kent) [inline src/containers/Dashboard.vue:179]: Why is this part of a component's data? It is not part of the Dashboard's state.

**Review by mirandayu131 — COMMENTED** [review]
I like the college name map. Do we also need to include SHA codes for hotelies?

File: src/containers/Dashboard.vue [hunk 2]
```diff
@@ -159,6 +159,24 @@ export default defineComponent({
...
+        ['cals', 'CALS'],
+        ['AG', 'CALS'],
+        ['EN', 'CoE'],
+        ['eng', 'CoE'],
+        ['HE', 'HumEc'],
+        ['humec', 'HumEc'],
+        ['IL', 'ILR'],
+        ['ilr', 'ILR'],
+        ['johnson', 'Dyson'],
+        ['BU', 'Dyson'],
```
[the remark this answers is not in the document]
Comment (jerrry1123) [reply src/containers/Dashboard.vue:178]: I couldn't find the place in the export toolcard where the pdf codes were being generated, so this is a temporary solution which we can fix later

**Pull request closed without merging**